docs/configuration.md

```diff
@@ -109,9 +119,9 @@ The host (hostname:port) of the cluster, http protocol is assumed. Deprecated, u
 The explicit version to use for this cluster.
 Define this to override the automatic version detection.
 
-**timeout** (number), default: 40000
+**timeout** (number)
 
-The timeout to set on the queries in ms.
+The timeout to set on the Druid queries in ms. See [documentation](https://druid.apache.org/docs/latest/querying/query-context.html)
```

Comment:
Should we document that there is no default value and by default Turnilo relies on Druid broker configuration?